docs/acp/user/app-manifest.md

```diff
@@ -210,6 +210,25 @@ For the manifest, make sure to follow these conditions:
 ```
 </details>
 
+<details>
+<summary>dependencies example</summary>
+
+```json
+"dependencies": {
+  "foo/bar": "^1.0.0",
+  "baz/qux": {
+    "version": "^1.0.0",
+    "compatibleBusinessModels": [
+      "B2B",
+      "B2C",
+      "B2B_MARKETPLACE",
+      "B2C_MARKETPLACE"
+    ]
```

Comment:
it's not the same as you have here https://spryker.atlassian.net/browse/APPS-5938?focusedCommentId=369907

Reply:
That's because here you have the manifest schema. In the screenshot attached to my comment there's a response body.